lib/tasks/dad_tasks.rake

```diff
@@ -244,12 +244,18 @@ namespace :dad_tasks do
       (1..3).each do |i|
         House.create(
           house_name: "House #{i}",
-          house_start: Time.now.utc.at_beginning_of_month.to_date,
+          house_start: Time.now.utc.at_beginning_of_month.to_date

```

Comment:
I forget. Is the rubocop-ism to have the comma, or not have the comma? Because I swear having the comma was the standard, but fuck if I know off the top of my head.